CHANGELOG.md

```diff
@@ -1,5 +1,46 @@
 ## Latest
 
+## 0.6.4
+
+### Important
+- Following components will be deprecated and archived in next release:
+  - clickhouse-cli-client
+  - clickhouse-grpc-client
+- No more builds for non-lts Java versions - no more Java 9 release builds.
```

Comment:
Isn't it a breaking change for users with the old Java version? If so, should we tag the release as `0.7.0`?

Reply:
Discussed. It will be in deprecations warning and will be removed in 0.7.0 release.